python/pyspark/tests/test_rdd.py

```diff
@@ -615,11 +615,11 @@ def test_repartitionAndSortWithinPartitions_desc(self):
     def test_repartition_no_skewed(self):
         num_partitions = 20
         a = self.sc.parallelize(range(int(1000)), 2)
-        l = a.repartition(num_partitions).glom().map(len).collect()
-        zeros = len([x for x in l if x == 0])
+        lst = a.repartition(num_partitions).glom().map(len).collect()
+        zeros = len([x for x in lst if x == 0])
```

Comment:
We iterate with `x`, we could just name it `xs`, but I am not insisting on that :)